sample/Kmd.Logic.Identity.Authorization.Sample/appsettings.json

```diff
@@ -0,0 +1,5 @@
+﻿{
+  "ClientId": "",
+  "ClientSecret": "",
+  "AuthorizationScope": ""
```

Comment:
Should we also add AuthorizationTokenIssuer here too? Just to make it clear you can override it if you want.

Reply:
Yes we could but I specifically say in the XML comments that you shouldn't.